packages/marketplace/src/settings.json

```diff
@@ -1,5 +1,8 @@
 {
-  "displayName": "Marketplace",
+  "displayName": {
```

Comment:
I inspired it from

https://github.com/spicetify/spicetify-cli/blob/master/CustomApps/lyrics-plus/manifest.json#L2-L9

Maybe we can add a `title` field? It will be still better to have menu item localized.